Formula/aescrypt.rb

```diff
@@ -19,13 +19,10 @@ def install
 
   test do
     (testpath/"key").write "kk=12345678901234567890123456789abc0"
-
-    require "open3"
-    Open3.popen3("#{bin}/aescrypt", "-k", testpath/"key") do |stdin, stdout, _|
-      stdin.write("hello")
-      stdin.close
-      # we can't predict the output
-      stdout.read.length > 0
-    end
+    original_text = "hello"
+    cipher_text = pipe_output("echo #{original_text} | #{bin}/aescrypt -k #{testpath}/key -s 128").chomp
+    deciphered_text = pipe_output("echo #{original_text} | #{bin}/aescrypt -k #{testpath}/key -s 128 | #{bin}/aesget -k #{testpath}/key -s 128").chomp
```

Comment:
Similar story here and I think it would be better to directly use the output of the previous line instead of running the unencrypted text again through `aescrypt`:

```ruby
deciphered_text = pipe_output("#{bin}/aesget -k #{testpath}/key -s 128", cipher_text)
```